src/hotspot/share/gc/shared/cardTable.hpp

```diff
@@ -172,6 +173,9 @@ class CardTable: public CHeapObj<mtGC> {
   // Resize one of the regions covered by the remembered set.
   void resize_covered_region(MemRegion new_region);
 
+  // Resize both of the regions covered by the remembered set.
+  void resize_covered_region_in_shared_virtual_space(MemRegion new_heap_region0, MemRegion new_heap_region1);
```

Comment:
Should this be "covered_regions", i.e., plural?

Reply:
> Should this be "covered_regions", i.e., plural?

It is just 1 region from the point of view of the card table. I think my comment here is what I should update. I'll fix the comment in the upstream PR.